hnn_core/network_builder.py

```diff
@@ -562,6 +568,9 @@ def aggregate_data(self, n_samples):
                 nrn_dpl = self._nrn_dipoles[_long_name(cell.name)]
                 nrn_dpl.add(cell.dipole)
 
+                if self.net._params['record_dcell']:
```

Comment:
okay so `record_dcell` does nothing to the neuron simulation ... the dipoles are recorded no matter what. The only difference is whether they are converted from neuron into python or not ... 

you may have opened a [pandora's box](https://github.com/jonescompneurolab/hnn-core/issues/669). If you want to go down that road, we can provide `cell_response.dipole` ... and then in a later pull request deprecate `simulate_dipole` to simply `simulate`

Reply:
How about I switch this to a `WIP` flag and we use this as a PR to work out the simulate_dipole API? I agree this is a natural place to make this happen, but the pandora's box may need to stay half open until COSYNE abstracts are submitted :wink:

Reply:
I agree with @jasmainak: cell-level dipoles should be accessible to the user via a similar API as other cell-level outputs.

@ntolley, WDYT of hashing out an improved simulated_dipole API with existing features first in a separate MAINT PR and keep this an ENH PR. I'm imagining this getting super huge and unwieldy....